src/payment/payment-action-creator.ts

```diff
@@ -60,6 +64,23 @@ export default class PaymentActionCreator {
         };
     }
 
+    generatePaymentIntent(
```

Comment:
Reason behind why we added this new endpoint is because existing one handles ClientToken only, on this one we handle PaymentIntents which maps a different set of data (CustomerId, BillingAddress, CurrencyCode, GrandTotal, etc.), on the other hand, we designed this endpoint to communicate directly SDK and BigPay because we there’s no need to store the token in memory.